lib/membrane/rtp/vad.ex

```diff
@@ -132,11 +132,11 @@ defmodule Membrane.RTP.VAD do
         handle_vad(buffer, rtp_timestamp, level, state)
 
       rollover == :next ->
-        {:ok, state} = handle_init(state)
-        {{:ok, buffer: {:output, buffer}}, state}
+        {[], state} = handle_init(ctx, state)
```

Comment:
This context is not used, and if `handle_init` matched on a struct it would crash (each callback has a different struct)
So either pass an empty map here or create `state_init` private function that will be called here.